src/economy/demographics.cpp

```diff
@@ -1482,6 +1482,25 @@ float get_estimated_type_change(sys::state& state, dcon::pop_id ids) {
 			: (std::ceil(demotion_chance * state.defines.promotion_scale * current_size)));
 }
 
+void save_yesterday_pop_size(sys::state& state, dcon::nation_id nation)
+{
+	for(auto prov : state.world.nation_get_province_ownership(nation)) {
+		for(auto pop : prov.get_province().get_pop_location()) {
+			state.world.pop_set_yesterday_size(pop.get_pop(), state.world.pop_get_size(pop.get_pop()));
```

Comment:
there is more than one pop in a province, so what this will do is essentially save the size of one of the pops living in the province, chosen at more or less random